cpu/native/can/candev_linux.c

```diff
@@ -7,7 +7,7 @@
  */
 
 /**
- * @ingroup     native_cpu
+ * @ingroup     candev_linux
```

Comment:
If one keeps the two, it results in a very weird rendering of the parent groups breadcrumbs. I think removing is one of the best solution.
Maybe I can add a ref to native_cpu: `@see native_cpu` ?